test/e2e/common/host_path.go

```diff
@@ -133,6 +133,7 @@ func mount(source *api.HostPathVolumeSource) []api.Volume {
 func testPodWithHostVol(path string, source *api.HostPathVolumeSource) *api.Pod {
 	podName := "pod-host-path-test"
 
+	priv := true
```

Comment:
I would consider those volumes tests in a different class (they need
privileged to do thinks like mounting). You shouldn't need to be privileged
just to get a hostMount to work.

On Fri, Aug 26, 2016 at 3:20 PM, Avesh Agarwal <notifications@github.com>
wrote:

> In test/e2e/common/host_path.go
> <https://github.com/kubernetes/kubernetes/pull/31542#discussion_r76473301>
> :
>
> > @@ -133,6 +133,7 @@ func mount(source *api.HostPathVolumeSource) []api.Volume {
> >  func testPodWithHostVol(path string, source *api.HostPathVolumeSource) *api.Pod {
> >  	podName := "pod-host-path-test"
> >
> > +	priv := true
>
> I just noticed that no test is doing that. See test/e2e/volumes.go too.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/31542/files/d5a5fd68e797f42ab582ac9964750b0277b27a22#r76473301>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAABYjJ857PXzBoIn12cs-R_3N6gJ6DTks5qjzyEgaJpZM4JuYK8>
> .
>